composer.json

```diff
@@ -0,0 +1,92 @@
+{
+  "name": "humanmade/core-sitemaps",
+  "description": "Core Sitemaps project",
+  "type": "wordpress-plugin",
+  "license": "GPL-2.0-or-later",
+  "config": {
+    "preferred-install": "dist",
+    "sort-packages": true,
+    "autoloader-suffix": "csm",
+    "platform": {
+      "php": "7.3"
+    },
+    "process-timeout": 600,
+    "vendor-dir": "vendor"
+  },
+  "repositories": [
+    {
+      "type": "composer",
+      "url": "https://wpackagist.org"
+    }
+  ],
+  "scripts": {
+    "post-install-cmd": [
+      "@setup"
+    ],
+    "post-update-cmd": [
+      "@setup"
+    ],
+    "setup": [
+      "@composer run-script --list"
+    ],
+    "local-tests": [
+      "@test:phpunit-local",
+      "@test:behat-local"
+    ],
+    "test:phpunit": [
+      "vendor/bin/phpunit --verbose --colors=always"
+    ],
+    "test:phpunit-local": [
+      "wp @local db query --dbuser=root --dbpass=password \"CREATE DATABASE IF NOT EXISTS wordpress_test DEFAULT CHARSET utf8\"",
+      "vagrant ssh -c \"cd /vagrant/content/plugins/core-sitemaps && WP_TESTS_DB_PASS=password composer run test:phpunit\""
+    ],
+    "test:phpcs": [
+      "vendor/bin/phpcs -nps --colors --report-code --report-summary --report-width=80 ."
+    ],
+    "test:behat-local": [
+      "# Make sure Selenium is running!",
+      "BEHAT_PARAMS='{\"extensions\":{\"Behat\\\\MinkExtension\":{\"base_url\":\"http://sitemaps.local\"},\"PaulGibbs\\\\WordpressBehatExtension\":{\"site_url\":\"http://sitemaps.local/wp\",\"wpcli\":{\"alias\":\"local\"}}}}' vendor/bin/behat --colors --format progress --"
```

Comment:
Hard coding the local URL fails if someone is using a different test domain (as I am). Here and below.

Reply:
Updated the code so someone with their site at a different address can run:
```
#hardcode
export WP_SITEURL=http://sitemaps.local 
composer run test:behat-local

# any vagrant install with wp cli
export WP_SITEURL=$(vagrant ssh -c "cd /vagrant && wp option get home") 
composer run test:behat-local
```